src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -368,7 +370,19 @@ private async Task<(ExcerptResult, SourceText)> ExcerptAsync(SourceText sourceTe
                 return (excerptResult, AbstractDocumentSpanEntry.GetLineContainingPosition(sourceText, documentSpan.SourceSpan.Start));
             }
 
-            private ImmutableDictionary<string, string> GetAggregatedCustomColumnsData(ImmutableDictionary<string, ImmutableArray<string>> customColumnsDataOpt)
+            private ImmutableDictionary<string, string> GetAggregatedCustomColumnsData(IEnumerable<KeyValuePair<string, ImmutableArray<string>>> customColumnsDataOpt, ImmutableArray<CodeAnalysis.AdditionalProperty> customColumns)
+            {
+                var result = GetAggregatedUsageColumnData(customColumnsDataOpt);
+
+                foreach (var column in customColumns)
+                {
+                    result = result.Add(column.propertyInfo.label, column.propertyInfo.value);
+                }
+
+                return result;
+            }
+
+            private ImmutableDictionary<string, string> GetAggregatedUsageColumnData(IEnumerable<KeyValuePair<string, ImmutableArray<string>>> customColumnsDataOpt)
```

Comment:
Make this a local function of `GetAggregatedCustomColumnsData` method added above and avoid `Usage` in the naming.